app/controllers/exhibits_controller.rb

```diff
@@ -7,6 +7,11 @@ def ingest_members
     ExhibitProxy.new(@exhibit).reindex
   end
 
+  def destroy
+    @exhibit.resources.destroy_all
```

Comment:
I'd have to do some testing to make sure dependent: :destroy_all will do what I want - the exhibit needs to exist in order for the solr IDs to be generated.